src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-sourceFolder.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>Specify the location of folders containing source code, relative to the project root folder. The specified folders and their subfolders are added to the top of the MATLAB search path. To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders.</p>
+    <p>Paste the source folder path in the File path box. To add more folders, click Add folder.</p>
```

Comment:
Thanks for the changes. We use boldface when mentioning UI text. So, a couple of changes are required in the second paragraph:

File path  -->  **File path**
Add folder --> **Add folder**

Reply:
Done